src/commons/progressLoader/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { BounceLoader } from 'react-spinners';
+
+const CircularProgressLoader = ({ loading }) => {
```

Comment:
Thanks for the good work @wasibani-roy , this file has some linting errors mostly no use of double quotes